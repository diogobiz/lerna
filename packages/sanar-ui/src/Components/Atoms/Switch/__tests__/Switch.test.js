import React from "react";
import TestRenderer from "react-test-renderer";

import ESSwitch from "../";

it("renders correctly", () => {
	const component = <ESSwitch />;

	const tree = TestRenderer.create(component).toJSON();
	expect(tree).toMatchSnapshot();
});
