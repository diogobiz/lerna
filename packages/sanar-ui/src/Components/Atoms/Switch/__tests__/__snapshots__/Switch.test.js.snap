// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  aria-checked={false}
  className="es-switch ant-switch"
  defaultChecked={false}
  onClick={[Function]}
  onKeyDown={[Function]}
  onMouseUp={[Function]}
  role="switch"
  type="button"
>
  <span
    className="ant-switch-inner"
  />
</button>
`;
